test/distributed/org/apache/cassandra/distributed/test/ReadDigestConsistencyTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.distributed.test;
+
+import java.util.Arrays;
+import java.util.UUID;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.apache.cassandra.distributed.Cluster;
+import org.apache.cassandra.distributed.api.ConsistencyLevel;
+import org.apache.cassandra.distributed.shared.DistributedTestBase;
+
+public class ReadDigestConsistencyTest extends TestBaseImpl
+{
+    public static final String TABLE_NAME = "tbl";
+    public static final String CREATE_TABLE = String.format("CREATE TABLE %s.%s (key int, s1 text static, c1 text, c2 text, c3 text, PRIMARY KEY (key, c1))", DistributedTestBase.KEYSPACE, TABLE_NAME);
+    public static final String INSERT = String.format("INSERT INTO %s.%s (key, s1, c1, c2, c3) VALUES (?, ?, ?, ?, ?)", DistributedTestBase.KEYSPACE, TABLE_NAME);
+
+    public static final String SELECT_C1 = String.format("SELECT key, c1 FROM %s.%s WHERE key = ?", DistributedTestBase.KEYSPACE, TABLE_NAME);
+    public static final String SELECT_C1_S1_ROW = String.format("SELECT key, c1, s1 FROM %s.%s WHERE key = ? and c1 = ? ", DistributedTestBase.KEYSPACE, TABLE_NAME);
```

Comment:
Nit: we don't need the `DistributedTestBase .` prefix, nor the import of `DistributedTestBase` above.

Reply:
ok